rotkehlchen/chain/evm/decoding/uniswap/v3/decoder.py

```diff
@@ -173,17 +139,51 @@ def _find_to_asset_and_amount(self, events: list['EvmEvent']) -> tuple[Asset, FV
 
     def _decode_deposits_and_withdrawals(self, context: DecoderContext) -> DecodingOutput:
         if context.tx_log.topics[0] == INCREASE_LIQUIDITY_SIGNATURE:
-            return self._maybe_decode_v3_deposit_or_withdrawal(
-                context=context,
-                event_action_type='addition',
+            is_deposit = True
+        elif context.tx_log.topics[0] == COLLECT_LIQUIDITY_SIGNATURE:
+            is_deposit = False
+        else:
+            return DEFAULT_DECODING_OUTPUT
+
+        position_id = int.from_bytes(context.tx_log.topics[1])
+        try:
+            # Returns a tuple containing information about the state of the LP position.
```

Comment:
nitpick: Docs or contract link to know where this info comes from. (I know you just moved it around)